source/structured-streaming.txt

```diff
@@ -200,6 +200,13 @@ more about continuous processing, see the `Spark documentation <https://spark.ap
 
 .. include:: /includes/fact-read-from-change-stream
 
+.. note::
+
+    Since structured streaming produces a single partition,
+    :ref:`partitioner configurations <partitioner-conf>` do not change the
+    behavior of read streams. This is true for both micro-batch processing and
```

Comment:
Suggestion:

I think it could be better to use [positive statements to make it easier to understand](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/) here.

I also think it could be helpful to add some information that makes the connection more clear as to why it doesn't change read streams behavior.

Question:

 Would it also be appropriate to include that the connector "ignores all partitioner configurations" (from the ticket description) since that seems like it might be a superset of read stream behavior?